engine/api/src/main/java/com/cloud/vm/VirtualMachineManager.java

```diff
@@ -61,6 +61,12 @@
     ConfigKey<Boolean> ResoureCountRunningVMsonly = new ConfigKey<Boolean>("Advanced", Boolean.class, "resource.count.running.vms.only", "false",
             "Count the resources of only running VMs in resource limitation.", true);
 
+    ConfigKey<Boolean> AllowExposeHypervisorHostnameAccoutLevel = new ConfigKey<Boolean>("Advanced", Boolean.class, "allow.expose.host.hostname",
+            "false", "If set to true, it allows the hypervisor host name on which the VM is spawned on to be exposed to the VM", true, ConfigKey.Scope.Account);
+
+    ConfigKey<Boolean> AllowExposeHypervisorHostname = new ConfigKey<Boolean>("Advanced", Boolean.class, "allow.expose.host.hostname",
```

Comment:
This is why I asked to remove the 'account' name from the configuration. It was only one configuration with scope = 'Account' so it could be overridden per account, and if not, the global value is used

Reply:
Ignore my comment, went through the requirements and both are needed